test/probot.test.ts

```diff
@@ -155,41 +155,40 @@ describe("Probot", () => {
     it("responds with the correct error if webhook secret does not match", async () => {
       expect.assertions(1);
 
-      probot.log.error = jest.fn();
+      // @ts-expect-error
+      probot.log.error = vi.fn();
       probot.webhooks.on("push", () => {
         throw new Error("X-Hub-Signature-256 does not match blob signature");
       });
 
       try {
         await probot.webhooks.receive(event);
       } catch (e) {
-        expect(
-          (probot.log.error as jest.Mock).mock.calls[0][1],
-        ).toMatchSnapshot();
+        expect((probot.log.error as Mock).mock.calls[0][1]).toMatchSnapshot();
       }
     });
 
     it("responds with the correct error if webhook secret is not found", async () => {
       expect.assertions(1);
 
-      probot.log.error = jest.fn();
+      // @ts-expect-error
+      probot.log.error = vi.fn();
       probot.webhooks.on("push", () => {
         throw new Error("No X-Hub-Signature-256 found on request");
       });
 
       try {
         await probot.webhooks.receive(event);
       } catch (e) {
-        expect(
-          (probot.log.error as jest.Mock).mock.calls[0][1],
-        ).toMatchSnapshot();
+        expect((probot.log.error as Mock).mock.calls[0][1]).toMatchSnapshot();
       }
     });
 
     it("responds with the correct error if webhook secret is wrong", async () => {
       expect.assertions(1);
 
-      probot.log.error = jest.fn();
+      // @ts-expect-error
+      probot.log.error = vi.fn();
```

Comment:
Just for clarification: Why dont we just cast it with `as LogFn` ?

Reply:
No reason, but I'll change it.